src/openrct2-ui/interface/Widget.cpp

```diff
@@ -413,7 +413,7 @@ static void widget_text(rct_drawpixelinfo* dpi, rct_window* w, rct_widgetindex w
         stringId = STR_STRING;
         formatArgs = &widget->string;
     }
-    gfx_draw_string_left_clipped(dpi, stringId, formatArgs, colour, l + 1, t, r - l);
+    gfx_draw_string_left_clipped(dpi, stringId, formatArgs, colour, { l + 1, r - 1 }, t);
```

Comment:
Seems like you swapped `y` and `width` here, as the signature was `int32_t x, int32_t y, int32_t width`. Shouldn't it be:
```suggestion
    gfx_draw_string_left_clipped(dpi, stringId, formatArgs, colour, { l + 1, t } r - 1);
```